ci/sawtooth-publish-js-sdk

```diff
@@ -0,0 +1,72 @@
+# Copyright 2017 Intel Corporation
```

Comment:
This copyright looks wrong for a new file.

Reply:
It's substantially copied from a file that was originally contributed by Intel on the 1.0 branch.